eng/pipelines/aggregate-reports.yml

```diff
@@ -32,6 +32,19 @@ jobs:
         artifact: packages
         patterns: '*'
 
+      - pwsh: |
+          npx cspell lint `
+            --config ./.vscode/cspell.json `
```

Comment:
Given cSpell out of the box looks for [other patterns](https://github.com/streetsidesoftware/cspell/blob/master/packages/cspell/README.md#customization), is there a reason we have to make this VSCode-specific and require every invocation to include the `--config` option? If we use default paths, people using VSCode can simply update the dictionary right from within VSCode as well.

Reply:
My original intent was to put a cspell.json file at the root of the repo. However this adds to the number of files that are at the root of the repo which could reasonably live elsewhere. Instead this is in the .vscode folder so that it [would be available to vscode](https://github.com/streetsidesoftware/vscode-spell-checker#customization) but not take up an additional row at the root of the repo. 

cc @weshaggard

Reply:
If that works with VSCode already, great; though, I still don't see how - if it didn't work by default - adding files to root is such a bad things. We have relatively few compared to a lot of repos. It's source code, not a foyer. :)

Reply:
Yeah I like to try my best to keep the root of the repo as clean as possible so I want to avoid random tool config files when we can. For this case I think it is reasonable pass the path to the tool in our usages and have it under the vscode directory where vscode will pick it up automatically. If we end up wanting to support something in VS as well then we can see what our options are.

Reply:
Right now there's no VS extension I can find that uses cSpell, but it would be nice. Might be worth investing a little time, or see if Mads is interested (he could probably whip one up in a few minutes). Even then, since VSCode helped set a precedent, seems reasonable to have a VS extension check .vscode/cspell.json as well.